### PR TITLE
Update pom with config server dependencies

### PR DESCRIPTION
**Pull request by Joshua-Mallory**

### CODE REVIEW

**Review by leandroyabut — COMMENTED** [review]
I just want to make sure looking through this that you didn't remove anything but added stuff is that correct? This is going to cause a lot of merge conflicts that whole left side is red.

**Review by Joshua-Mallory — COMMENTED** [review]
line 21, 37-40 and 96-106

File: user-microservice/pom.xml [hunk 1]
```diff
@@ -1,128 +1,143 @@
...
-            </plugin>
-        </plugins>
-    </build>
+	<groupId>com.aline</groupId>
+	<artifactId>user-microservice</artifactId>
+	<version>0.1.0</version>
+	<name>User Microservice</name>
+	<description>User microservice that handles CRUD operations on users.</description>
+	<properties>
+	<spring-cloud.version>2020.0.0</spring-cloud.version>
```
Comment (leandroyabut) [inline user-microservice/pom.xml:21]: Specified version number for spring cloud

**Review by leandroyabut — APPROVED** [review]
Looks good to me.

**Pull request merged**